examples/nextjs-tiptap-novel/src/lib/content.ts

```diff
@@ -0,0 +1,356 @@
+export const defaultEditorContent = {
```

Comment:
Could we tweak this? Especially the tweet and the Vercel Blob broken asset, but also maybe removing things like uploads if we disable them etc

Reply:
Yep! Meant to remove this altogether, as Tiptap doesn't ave an initial content option yet (WIP).